.github/workflows/nightly.yaml

```
@@ -0,0 +1,29 @@
+# Schedule to trigger Main workflow with cleared cache
+name: nightly
+on:
+  schedule:
+    - cron: '0 5 * * *'
+jobs:
+  clear-cache:
+    uses: ./.github/workflows/cache.yaml
+  main:
+    needs: [clear-cache]
+    uses: ./.github/workflows/main.yaml
+    secrets:
+      SONAR_TOKEN: { required: true }
```

```suggestion
      SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```
... and so on. This is annoyingly verbose but reusable workflow aren't necessarily in the same repo, so it make sense that we have to be explicit

---

Actually we can use 
`secrets: inherit` since they're in the same repo now/relatively recently:

https://github.blog/changelog/2022-05-03-github-actions-simplify-using-secrets-with-reusable-workflows/

---

I think inheritance is allowed so long as they're under the same org?

---

Great find, let's do this! That's a lot of yaml we can delete then